.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "dotnet-test-explorer.testProjectPath":"/tests/Imgix.Tests/Imgix.Tests.csproj"
+}
```

Comment:
Should this be checked in?

Reply:
Probably not 🤦🏼 

Debatably, we could add `dotnest-test-explorer` as a reccomended extension inside `.vscode`, and then leaving this in there would make sense since it would simplify the testing support for those who install it. But realistically, you're right that the less extension-specific stuff we check in the better.